components/Header/index.js

```diff
@@ -0,0 +1,7 @@
+export default function Header() {
+  return (
+    <header className="header">
+      <h1 className="header__title">Endstagram</h1>
```

Comment:
better use only styled components, don't mix them up with classes and css files

Reply:
Hi Nina, 
Thank you for your review! 😊 Damn, you're of course right, gonna change it! (I'm still a little stuck with the old methods of styling... 🤦‍♀️)